src/main/java/org/gridsuite/study/server/dto/modification/ModificationType.java

```diff
@@ -34,6 +33,7 @@ public enum ModificationType {
     DELETE_VOLTAGE_LEVEL_ON_LINE,
     DELETE_ATTACHING_LINE;
 
+    // TODO transfer method to the enum
```

Comment:
is it planned for another PR, or is it an oversight ?

Reply:
Yes  for another PR